scripts/local/common.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/sh
+
+
+# ======================================= FUNCTIONS START =======================================
+
+function stop_docker_container() {
+
+  [ -z "$DOCKER_HOST" ] && { echo "Environment variable DOCKER_HOST must be set"; exit 1; }
```

Comment:
Having issues setting DOCKER_HOST on my local mac using the Docker Toolbox.  According to instructions it should not be set: https://docs.docker.com/docker-for-mac/docker-toolbox/#setting-up-to-run-docker-for-mac.   Also we did not need to set the DOCKER_HOST for docker-compose when running the tests on the Jenkins CI machine.  So for now the docker commands fail to fire when following instructions in README :-(